.github/workflows/docker-build.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Docker build/push
       uses: docker/build-push-action@v2
       with:
-        platforms: linux/amd64,linux/arm/v7
+        platforms: linux/amd64,linux/arm/v7, linux/arm64
```

Comment:
```suggestion
        platforms: linux/amd64,linux/arm/v7,linux/arm64
```

Reply:
Could you additionally add the new architecture into the README.md? At the end, there is a list of supported architectures.

Reply:
Thanks, done!